sdk/src/beta9/abstractions/integrations/vllm.py

```diff
@@ -330,6 +330,12 @@ def __call__(self, *args: Any, **kwargs: Any):
                 f"{self.engine_config.download_dir}/{chat_template_filename}"
             )
 
+        if "HF_TOKEN" in os.environ:
+            hf_token = os.environ["HF_TOKEN"]
+            import huggingface_hub
```

Comment:
Should we check if huggingface_hub is installed e.g. do a try/except?

Reply:
I think we should be ok because it is a dependency of vllm. I can explicitly add it to the image requirements to make sure though

Reply:
Just saw your change. Can't that be overwritten? I think if we're not going to try/cache the import, then we need to make sure huggingface is installed during `__init__`.

Reply:
Installed meaning, calling add_python_packages in the init function.

Reply:
hmm yeah, I will change the logic to be similar to how we handle the volume mounts.

Reply:
Okay cool. Looks good. We just want to make sure the package is available or if its not, handle the potential exception.

Reply:
Yeah makes sense. When I thought about it, I concluded that we should make sure vllm is there no matter what.